sandbox/src/layer-manager/layer-manager.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react'
+import ReactMapGL from 'react-map-gl'
+import styles from './layer-manager.module.css'
+
+import LayerManager from '@globalfishingwatch/map-components/src/layer-manager'
+
+class MapPage extends Component {
+  state = {
+    mounted: false,
+    viewport: {
+      width: '100%',
+      height: '100%',
+      latitude: 37.7577,
+      longitude: -122.4376,
+      zoom: 8
+    },
+    layers: [
+      { id: 'north-star', type: 'basemap' },
+      {
+        id: 'mpant',
+        type: 'carto',
```

Comment:
I think this generator should be named 'context' /  'ContextGenerator'.
Carto seems to generic to me and to distant from the domain. Also, we are planning to have animated polygon layers in the future, which would be a very different kind of layer, but likely to use a Carto backend.